projects/arlas-map/src/lib/arlas-map-framework.service.ts

```diff
@@ -32,59 +33,75 @@ import { VectorStyle } from './map/model/vector-style';
 @Injectable({
   providedIn: 'root'
 })
-export abstract class ArlasMapFrameworkService {
+export abstract class ArlasMapFrameworkService<L, S, M> {
```

Comment:
I would add a description stating what each of the L, S, M types are just so that it is clear for outside people